sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFn.java

```diff
@@ -397,6 +397,47 @@ public Duration getAllowedTimestampSkew() {
     String value();
   }
 
+  /////////////////////////////////////////////////////////////////////////////
+
+  /**
+   * Annotation for declaring that a state parameter is always fetched.
+   *
+   * <p>A DoFn might not fetch a state value on every element, and for that reason runners may
+   * choose to defer fetching state until read() is called. Annotating a state argument with this
+   * parameter provides a hint to the runner that the state is always fetched. This may cause the
+   * runner to prefetch all the state before calling the processElement or processTimer method,
+   * improving performance. See the following code for an example:
```

Comment:
Add a comment that the annotation only works for ReadableState

Reply:
done